src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -430,7 +497,6 @@ const useGroupInfo = (channelId = null) => {
    * @param {('view' | 'remove' | 'message' | 'block' | 'message-anonymously')} status
    */
   const handleOpenPopup = async (status) => {
-    console.warn('selectedUser', JSON.stringify(selectedUser));
     if (status === 'view') {
       setOpenModal(false);
       handleOpenProfile(selectedUser).catch((e) => console.log(e));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The console warning statement has been removed which is good for production readiness. The function `handleOpenPopup` seems to be handling different statuses properly.
```diff
- console.warn('selectedUser', JSON.stringify(selectedUser));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->